assembly_pipeline_readfq/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.16.0
+
+#tag  microbiome-informatics/assembly-pipeline.readfq:latest
```

Comment:
It's missing some metadata, LICENCE, VERSION, LINK to the tool.
Also, the folder should be `readfq`.